docs/dependency-updates.md

```diff
@@ -215,31 +215,66 @@ TODO: Describe how the `fxa-rp-events` endpoint uses these packages.
 
 ### `uuid`
 
-TODO: Describe how to verify that `uuid` updates didn't break anything.
+Used in places where a random identifier is needed:
+
+- as a `nonce` in `src/middleware.ts` for CSP
+- server-side Glean document IDs in `src/app/functions/server/glean.ts`
+- as an experimentation ID for Nimbus in `src/app/functions/server/getUserId.tsx` and `src/app/functions/server/getExperimentationId.ts`
+- for email verification tokens in `src/db/tables/emailAddresses.js`
+
+1. Check that `nonce` is present in `content-security-policy` header for HTTP responses.
+2. Check that `experimentationId` is set as a cookie in the form `guest-${uuid}`
+3. Add a secondary email from the Settings screen and ensure that email verification works
 
 ### `@aws-sdk/*`
 
-TODO: Describe how to verify that AWS SDK updates didn't break anything.
+Used for S3, by the `src/scripts/s3.js` cron job.
```

Comment:
```suggestion
Used for S3, by the `npm run cron:db-pull-breaches` cron job.
```

Weird that `s3.js` is in `/src/scripts/`, since it seems to only export a utility function `uploadToS3` - I don't see it being run as a cronjob itself.

So then I suppose the way to verify this is to run `npm run dev:cron:db-pull-breaches` locally and check that it completes without errors?

Reply:
Hm yeah I think `s3.js` is just a utility library essentially, it should probably be moved to `src/utils` maybe...  thanks for suggestion, that sgtm.

Reply:
If I prefix `dev:` (to not have to compile), it does work. You might need these env vars though (also listed in `.env.local.example`):

```
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
```